src/main/scala/org/allenai/pipeline/Workflow.scala

```diff
@@ -25,6 +25,64 @@ case class Workflow(nodes: Map[String, Node], links: Iterable[Link]) {
   def errorNodes() = nodes.filter {
     case (nodeId, node) => node.outputMissing
   }
+
+  lazy val renderHtml: String = {
```

Comment:
Nit: I have a preference for either val or def. lazy val is cute, but also confusing, as I keep asking myself why it's not a val or def? :)